packages/lesswrong/lib/collections/posts/views.js

```diff
@@ -550,7 +550,6 @@ Posts.addView("sunshineNewPosts", function () {
     selector: {
       reviewedByUserId: {$exists: false},
       frontpageDate: viewFieldNullOrMissing,
-      authorIsUnreviewed: null
```

Comment:
What testing have you done on this change? The last few things in this reference class had surprising side-effects.